app/assets/stylesheets/navody-frontend-fixes.scss

```diff
@@ -40,3 +40,8 @@ a.sdn-timeline-dropdown__option {
   padding: 25px 30px 10px 30px;
   background-color: #e5f2e4;
 }
+
+/* TODO move to frontend code */
+.sdn-next-link__link {
+  display: block;
```

Comment:
Preco inak vo fronende to nevyzera zle?  https://frontend.navody.digital/custom-components/next-link

Reply:
vyzera to zle aj tu, je to zalomene na 2 riadky. Myslim, ze je to lepsie ked ten sublink bude v jednom riadku, samozrejme nemusi

Reply:
Inak. 
![image](https://user-images.githubusercontent.com/26342/68197065-de974900-ffb9-11e9-895d-74546b2d6839.png)

Takto to vyzerat ma. Ale nechapem preco ta sipka je na produkcii nevycentrovana.

Reply:
u mna to na frontende vyzera takto, preto ta zmena na display:block
<img width="387" alt="Screenshot 2019-11-05 at 11 00 42" src="https://user-images.githubusercontent.com/2971007/68198135-bf012000-ffbb-11e9-9ad8-580257c55f4a.png">